ledger/participant-integration-api/src/main/scala/platform/indexer/ha/KillSwitchCaptor.scala

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.indexer.ha
+
+import java.util.concurrent.atomic.AtomicReference
+
+import akka.stream.KillSwitch
+import com.daml.logging.{ContextualizedLogger, LoggingContext}
+
+/** A KillSwitch which captures it's usage
+  * - Shutdown always wins
+  * - From aborts, the last abort wins
+  * - With setDelegate() we can set a delegate KillSwitch, which to usage will be replayed
```

Comment:
Can you rephrase this comment. It is not clear

Reply:
rephrased